Sample_v3/Extensions/ChannelMember+Extensions.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+
+/// Creates formatted string for channel member online status.
+///
+/// Example result: `online` or `4 min ago`
+func createMemberOnlineStatusInfoString(for member: ChatChannelMember) -> String? {
+    if member.isOnline {
+        return "online"
+    } else {
+        return member.lastActiveAt?.timeAgo
+    }
+}
+
+/// Creates formatted string for channel member name and ban status.
+///
+/// Example result: `Luke Skywalker` or `Luke Skywalker 🚫`
```

Comment:
... or `Luke Skywalker (You)` for the current user.

Reply:
fixed in e0456857327a8ac49840441c24f7d2216cccb305